module-2/backend/week_two.md

```diff
@@ -25,20 +32,32 @@ end
 ```
 
 Now how would you find the owner of the team with an id of 4?
+Team.owners.find(4)
```

Comment:
We need to find the team first.

```ruby
Team.find(4).owner
```

Be very careful here: `owner` is an **instance** method. You are calling it on the **class**. The Team class doesn't have an owner. A specific team does.